domain/ide/converter/converter.go

```diff
@@ -52,30 +52,44 @@ func ToCodeActions(issues []snyk.Issue) (actions []lsp.CodeAction) {
 }
 
 func ToCodeAction(issue snyk.Issue, action snyk.CodeAction) lsp.CodeAction {
+	var id *lsp.CodeActionData = nil
+	if action.Uuid != nil {
+		i := lsp.CodeActionData(*action.Uuid)
+		id = &i
+	}
 	return lsp.CodeAction{
 		Title:       action.Title,
 		Kind:        lsp.QuickFix,
 		Diagnostics: ToDiagnostics([]snyk.Issue{issue}),
 		IsPreferred: action.IsPreferred,
 		Edit:        ToWorkspaceEdit(action.Edit),
 		Command:     ToCommand(action.Command),
+		Data:        id,
 	}
 }
 
-func ToCommand(command snyk.Command) sglsp.Command {
-	return sglsp.Command{
+func ToCommand(command *snyk.Command) *sglsp.Command {
+	if command == nil {
+		return nil
+	}
+
+	return &sglsp.Command{
 		Title:     command.Title,
 		Command:   command.CommandId,
 		Arguments: command.Arguments,
 	}
 }
 
-func ToWorkspaceEdit(edit snyk.WorkspaceEdit) sglsp.WorkspaceEdit {
+func ToWorkspaceEdit(edit *snyk.WorkspaceEdit) *sglsp.WorkspaceEdit {
+	if edit == nil {
```

Comment:
This seems to change the convert semantics to always return non-nil here.

Reply:
To be clear: It's ok for me to return `nil` here, but it may be a regression.

Reply:
If `edit` is `nil` we return `nil`, this is ok